src/endpoints/one_to_all.cc

```diff
@@ -20,14 +20,17 @@ namespace motis::ep {
 
 namespace n = nigiri;
 
-constexpr auto const kMaxResults = 65535U;
-constexpr auto const kMaxTravelMinutes = 90U;
-
 api::Reachable one_to_all::operator()(boost::urls::url_view const& url) const {
+  auto const max_travel_minutes =
+      config_.timetable_
+          .and_then([](config::timetable const& x) {
```

Comment:
`one_to_all` endpoint can only be instantiated if `tt_` is available (as it's a reference, not a pointer in the member variable). So `config_.timetable_` has to be set. If in doubt add a `utl::verify`.